### PR TITLE
added 10  new terms

### PR DESCRIPTION
**Pull request by Mide001**
added the following terms: 
Vaporware
Transactions per second (TPS)
Total Value Locked (TVL)
satoshi
Scalability
rekt
PFP
floor
play to earn
shill

### CODE REVIEW

File: data/terms.json [hunk 1]
```diff
@@ -7,6 +7,10 @@
     "term": "bitcoin",
     "meaning": "Satoshi Nakamoto, a pseudonym, developed the first decentralized, peer-to-peer digital money in 2009. Unlike government-issued currencies, Bitcoin boasts lower transaction fees than existing online payment options and is managed by a decentralized authority."
   },
+  {
+    "term": "satoshi",
+    "meaning": "the smallest denomination of BTC, equal to 0.00000001 bitcoin. Satoshis are named after Bitcoin’s pseudonymous creator, Satoshi Nakamoto."
```
Comment (Olanetsoft) [inline data/terms.json:12]: Thank you for your awesome contribution.

I have some feedback.

- Add the name of the term in the meaning eg. 
`Satoshi is the smallest denomination ....  ` not `the smallest demonination ...`

File: data/terms.json [hunk 2]
```diff
@@ -39,6 +43,14 @@
     "term": "decentralized application",
     "meaning": "A Dapp is a blockchain-based application built with open-source technology. Dapps live independently of centralized groups or figures, and they frequently use awarded tokens to incentivize users to keep them up to date."
   },
+  {
+    "term": "vaporware",
+    "meaning": "A product or project that is announced and marketed but never actually materializes."
```
Comment (Olanetsoft) [inline data/terms.json:48]: Same here

File: data/terms.json [hunk 3]
```diff
@@ -87,6 +103,10 @@
     "term": "smart contract",
     "meaning": "A smart contract is a self-executing code that is deployed on a blockchain. Smart contracts eliminate the requirement for a central authority, legal system, or external enforcement mechanism to carry out trustworthy transactions and agreements between distant, anonymous participants."
   },
+  {
+    "term": "shill",
+    "meaning": "a person is engaged in advertising a cryptocurrency or NFT project by endorsing the project or product in public forums or communities."
```
Comment (Olanetsoft) [inline data/terms.json:108]: Same here

File: data/terms.json [hunk 4]
```diff
@@ -203,14 +223,26 @@
     "term": "sharding",
     "meaning": "Sharding is a technique for breaking down a network's nodes into smaller groups (shards) to improve scalability. These shards can then reach a consensus on behalf of the entire network, eliminating the requirement for each node to execute each transaction."
   },
+  {
+    "term": "scalability",
+    "meaning": "a protocol’s capacity to handle higher demand and increase transaction throughput as the network grows"
```
Comment (Olanetsoft) [inline data/terms.json:228]: Same here

File: data/terms.json [hunk 5]
```diff
@@ -203,14 +223,26 @@
...
+  },
   {
     "term": "rug pull",
     "meaning": "Rug pulling is a scam technique in which a crypto enterprise takes the monies invested in its protocol and runs with them. If someone can sell a significant amount of the circulating supply, the supply will rapidly expand, causing the asset's price to drop."
   },
+  {
+    "term": "rekt",
+    "meaning": "as in “wrecked,” used to express that one has suffered a huge loss."
```
Comment (Olanetsoft) [inline data/terms.json:236]: same here

File: data/terms.json [hunk 6]
```diff
@@ -203,14 +223,26 @@
...
+  },
   {
     "term": "protocol",
     "meaning": "A program's basic software layer is called a protocol. A protocol has become a catch-all phrase for both layer one blockchain networks and the layer two applications developed on top of them. Protocols include Bitcoin, Ethereum, Uniswap, and Lightning Network."
   },
+  {
+    "term": "play to earn",
+    "meaning": "a business model for the video game industry that allows players to earn digital assets they can sell for money in a marketplace. Many of the assets are NFTs, such as a game skin, cool weapon, or an animal to breed. Examples include Axie Infinity, ZED RUN, Splinterlands, and Cryptofighters."
```
Comment (Olanetsoft) [inline data/terms.json:244]: Same here

File: data/terms.json [hunk 7]
```diff
@@ -527,6 +563,10 @@
     "term": "Ethereum Request for Comments",
     "meaning": "Ethereum Request for Comments is the standard smart contract outline on which Ethereum-based smart contracts are built."
   },
+  {
+    "term": "floor",
+    "meaning": "the floor or floor price is the lowest price to buy an NFT on the secondary market. It’s the most popular metric for tracking an NFT collection’s performance over time and comparing it to others."
```
Comment (Olanetsoft) [inline data/terms.json:568]: Start with a capital letter

**Review by Olanetsoft — CHANGES_REQUESTED** [review]
Looking forward to those changes @Mide001 

Let's get this merged so that we can proceed with other fixes.

**Comment by Mide001** [comment]
Alright

**Pull request merged**